x-pack/heartbeat/monitors/browser/synthexec/synthexec.go

```diff
@@ -30,7 +30,7 @@ const debugSelector = "synthexec"
 func SuiteJob(ctx context.Context, suitePath string, params common.MapStr, extraArgs ...string) (jobs.Job, error) {
 	// Run the command in the given suitePath, use '.' as the first arg since the command runs
 	// in the correct dir
-	newCmd, err := suiteCommandFactory(suitePath, append(extraArgs, ".")...)
+	newCmd, err := suiteCommandFactory(suitePath, append([]string{suitePath}, extraArgs...)...)
```

Comment:
is this a bug?

Reply:
Totally, no worries. Yes this is the issue with varidiac args in the `commander` library we are using - This is one of the reason why I was suggesting to add a test here - https://github.com/elastic/beats/pull/25808#discussion_r646986725

Glad we caught the error. If you are curios - https://github.com/tj/commander.js/blob/master/docs/options-taking-varying-arguments.md